modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -358,13 +358,11 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
             ///////////////////////////////////////////////////////////////////////
             ////////////// Make sure the file name matches the format   ///////////
             ///////////////////////////////////////////////////////////////////////
-            if (($file_name != $pscid."_". $candid ."_". $visit_label . ".zip")
-                && ($file_name != $pscid."_". $candid ."_". $visit_label . ".tgz")
-                && ($file_name != $pscid."_". $candid ."_". $visit_label . ".tar.gz")
-            ) {
```

Comment:
alternatively you could use a regular expression. Something likelike:

if preg_match(/^{$pscid}_{$candid}_{$visit_label}.*(zip|tgz|tar\.gz/) {
}